dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1006,6 +1007,11 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey UNDERFS_CHECKSUM_TYPE =
+      enumBuilder(Name.UNDERFS_CHECKSUM_TYPE, ChecksumType.class)
+          .setDefaultValue(ChecksumType.MD5)
+          .setDescription("UFS checksum type.")
+          .build();
```

Comment:
do u need to set Scope? does it matter?

Reply:
I think scope is not really used except few checks